src/models/Host.js

```diff
@@ -47,7 +47,7 @@ class Host {
 
     @JsonProperty()
     @JsonElementType(String)
-    languagesSpoken = ["pl"];
+    languagesSpoken = [];
```

Comment:
Oh cmon. This is a proper assumption. Can't think of host who does not speak at least their own language ;)

Reply:
That is for sure :) But I've thought the freshly created model should have empty values and not contain any business assumptions like this.